python/ctsm/site_and_regional/modify_singlept_site_neon.py

```diff
@@ -550,7 +502,7 @@ def main():
 
     # --  directory structure
     current_dir = os.getcwd()
-    parent_dir = os.path.dirname(current_dir)
+    # parent_dir = os.path.dirname(current_dir)
```

Comment:
...if not used, I would delete.
Same comment for obs_top a few lines down.

Reply:
Same possibly for a few print statements further down.

Reply:
Good points. There are a lot of print statements in this file. @wwieder , it looks like you had some TODO's related to the print sections that maybe got resolved? Do you want these print statements still? I'm personally not sure what's relevant/useful for folks looking at soil levels.

Reply:
Looks like these were from Negin [NS]. I'm not sure if we should keep them or not?

Reply:
Ah, yes, thanks @wwieder , I do see some of these comments are from Negin, too. I think there's the bit marked "TODO Will: if I sum them up , are they 3.5? (m) YES" and some discussion on updates that should instead be made to subset_data-- were these resolved?